### DISCUSSION TITLE
shell_bone and fire_bone2

### DISCUSSION THREAD

**Discussion by romanpopulik** [OGSR/OGSR-Engine#310] [Ideas]
Is it possible to add shell_bone and fire_bone2 just like in COP ? 

Also, can anyone explain me, what exactly below does:

> Новые опции для конфигов оружия: scope_bone, silencer_bone, launcher_bone: для переопределения имен костей при установленных аддонах.

**Comment by joye-ramone** [comment]
Using scope_bone, silencer_bone, launcher_bone you can define custom name for bones that engine will use for scope, silencer and gl.

**Reply by romanpopulik** [reply]
So when I'm porting weapon form one game to OGSR, I don't have to change scope, silencer and launcher bones names in hud model to wpn_scope, wpn_silencer and wpn_launcher because I can define them in config file ? Do I understand it right ?

**Reply by joye-ramone** [reply]
yep

**Comment by romanpopulik** [comment]
How actually fire_bone works in OGSR (SOC) ? In COP, muzzle effect appears in the exact place where fire_bone is placed. But in OGSR, when it set 0,0,0 values for fire_bone, muzzle effect is misaligned. Moreover, the place of effect changes in relation to weapon hud model, when Im changing rotation weapon position values.
